tool/python/examples/train_cifar10.py

```diff
@@ -28,27 +28,49 @@
 import numpy as np
 
 current_path_ = os.path.dirname(__file__)
-singa_root_=os.path.abspath(os.path.join(current_path_,'../../..'))
+singa_root_ = os.path.abspath(os.path.join(current_path_,'../../..'))
 sys.path.append(os.path.join(singa_root_,'tool','python'))
 
 from singa.driver import Driver
 from singa.layer import *
 from singa.model import *
-from singa.utils.utility import swap32
 
-fname_mean_image = 'tool/python/examples/datasets/cifar10_mean_image'
-mean_image = np.fromfile(fname_mean_image)
+dataset_dir_ = singa_root_ + "/examples/cifar10/cifar-10-batches-bin"
+mean_image = None
+
+def compute_mean_image():
```

Comment:
There are two different versions of the cifar10 dataset.
If we use the python version, then the code would be simpler
```
def unpickle(file):
    import cPickle
    fo = open(file, 'rb')
    dict = cPickle.load(fo)
    fo.close()
    return dict

cifar10=unpickle(file)
image=cifar10['data'].astype(dtype=float)
label=np.asarray(cifar10['label'], dtype=int)
mean=np.average(image, axis=0)
image=image-mean[